src/main/java/org/jfrog/hudson/pipeline/common/types/deployers/GradleDeployer.java

```diff
@@ -29,18 +29,20 @@
     @Override
     @JsonIgnore
     public ServerDetails getDetails() throws IOException {
+        RepositoryConf releaseRepositoryConf = new RepositoryConf(repo, repo, false);
+        if (server == null) {
+            // No deployer
+            return new ServerDetails("", "", releaseRepositoryConf, null, releaseRepositoryConf, null, "", "");
```

Comment:
I think that it would be simpler in this case to simply return null, and make sure all usages of this method can handle it.
If we do this, the first line of this method can be moved be after this null check.